packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -144,6 +144,9 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
   // Invoked once, only on the client (not on the server), immediately after the initial rendering occurs.
   public componentDidMount() {
     this._events.on(this._targetWindow, 'resize', this.dismiss);
+    if (this.props.onMenuOpened) {
```

Comment:
This will get called after the Layer's componentDidMount is called, which is appropriate. Since the Layer moves it's contents then it mounts, and then it's children mount. When the contextualmenu's componentDidMount has been called that means that it has rendered and should be ready to interact with.